filebeat/autodiscover/builder/hints/logs.go

```diff
@@ -22,9 +24,13 @@ const (
 	excludeLines = "exclude_lines"
 )
 
+// validModuleNames to sanitize user input
+var validModuleNames = regexp.MustCompile("[^a-zA-Z0-9]+")
```

Comment:
We should maybe include hyphen and/or underscore, we have a case in metricbeat.

Reply:
This only filters modules for filebeat, normally we wouldn't need to check this, but as this will go to the disk to retrieve module YAML we want to make sure people doesn't play with this value.

I'm ok with adding a check in a follow up PR, although it's not really needed by now

Reply:
Ok to add it in a follow-up PR